### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by KreppelKlatscher** [aminalaee/sqladmin#479] [General]
Is there a opportunity to include some custom css for the given templates or how can i easly include a template, whch uses the given templates but has some custom css files defined. 
In detail i want to adjust the size of the logo at the admin page. Currently i doing this by modifing the .navbar-brand-image css-class in tabler.min.css but im looking for a cleaner way.
Thanks everbody

**Comment by aminalaee** [comment]
Hi, I will add a section to the documentation soon.

**Comment by johanboekhoven** [comment]
In the admin main page I've added a new Mount like so:

`
admin.app.router.routes.append(
    Mount("/extras", StaticFiles(directory="./app/extras"), name="extras")
)
`

Then in your templates you can call it like

`
    <link rel="icon" type="image/x-icon" href="/extras/favicon.ico">
`

I'm sure there are more ways to do it.
